### ISSUE TITLE
Completions crash detected on DefinitelyTyped

### ISSUE THREAD

**Issue by andrewbranch** [microsoft/TypeScript#35632]
```
TypeError: Cannot read property 'kind' of undefined
    at tryGetClassLikeCompletionSymbols (/home/vsts/.dts/typescript-installs/3.7/node_modules/typescript/lib/typescript.js:108084:47)
    at tryGetGlobalSymbols (/home/vsts/.dts/typescript-installs/3.7/node_modules/typescript/lib/typescript.js:107515:24)
    at getCompletionData (/home/vsts/.dts/typescript-installs/3.7/node_modules/typescript/lib/typescript.js:107315:22)
    at Object.getCompletionsAtPosition (/home/vsts/.dts/typescript-installs/3.7/node_modules/typescript/lib/typescript.js:106679:34)
    at Object.getCompletionsAtPosition (/home/vsts/.dts/typescript-installs/3.7/node_modules/typescript/lib/typescript.js:131279:35)
    at getCompletionsAtPosition (/home/vsts/work/1/s/node_modules/@andrewbranch/definitely-not-slow/src/measure/measureLanguageServiceWorker.ts:98:39)
    at measureAtPosition (/home/vsts/work/1/s/node_modules/@andrewbranch/definitely-not-slow/src/measure/measureLanguageServiceWorker.ts:83:5)
    at /home/vsts/work/1/s/node_modules/@andrewbranch/definitely-not-slow/src/measure/measureLanguageServiceWorker.ts:61:8
    at Generator.next (<anonymous>)
    at /home/vsts/work/1/s/node_modules/@andrewbranch/definitely-not-slow/dist/src/measure/measureLanguageServiceWorker.js:8:71

Failed measurement on request: {
  "fileName": "/home/vsts/work/1/s/types/sharepoint/sharepoint-tests.ts",
  "start": 62930,
  "packageDirectory": "/home/vsts/work/1/s/types/sharepoint",
  "tsPath": "/home/vsts/.dts/typescript-installs/3.7/node_modules/typescript"
}
```

https://github.com/DefinitelyTyped/DefinitelyTyped/pull/40963

**Comment by andrewbranch** [comment]
Also on https://github.com/DefinitelyTyped/DefinitelyTyped/pull/46023/files

**Comment by minestarks** [comment]
The fuzzer also found one: https://github.com/microsoft/tsserverfuzzer/issues/254

**Comment by sheetalkamat** [comment]
Should this be fixed by #39731

**Comment by andrewbranch** [comment]
@sheetalkamat it looks like the fuzzer reproduced it with nightlies after #39731 was already merged.
